app/src/lib/feature-flag.ts

```diff
@@ -78,11 +78,6 @@ export function enableCheckoutCommit(): boolean {
   return true
 }
 
-/** Should ci check runs show logs? */
-export function enableCICheckRunsLogs(): boolean {
-  return false
-}
```

Comment:
It'd be nice to get errors/warnings/something for exported functions that aren't used. I see some options in https://github.com/typescript-eslint/typescript-eslint/issues/371 not sure if it'd be worth checking!